seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
+    """ForwardModel Class:
+
+    Abstract base class for the SEIR model class.
+
+    Parameters
```

Comment:
The class itself doesn't actually have any input parameters, i.e. you will instantiate the class with
`seirmo.ForwardModel()` and not with `seirmo.ForwardModel(parameters)`. You can see this by looking at the `__init__` method.

Reply:
So could you please remove this from the class doc string.